src/declarations/contracts/telemetryTypes/metricTelemetry.ts

```diff
@@ -7,7 +7,7 @@ import { Telemetry } from "./telemetry";
  */
 export interface MetricTelemetry extends Telemetry {
     /** List of metrics. Only one metric in the list is currently supported by Application Insights storage. If multiple data points were sent only the first one will be used. */
-    metrics: MetricPointTelemetry[];
+    metrics?: MetricPointTelemetry[];
```

Comment:
Made metrics optional here and updated the typing accepted by trackMetric to `MetricTelemetry & MetricPointTelemetry` such that customers can pass MetricPointTelemetry including additional `properties`. However, this means that we need to make `metrics` optional since coming from AppInsights 2.x customers are not expecting to pass a MetricPointTelemetry array.